dex/src/matching.rs

```diff
@@ -529,7 +546,9 @@ impl<'ob> OrderBookState<'ob> {
 
         self.market_state.referrer_rebates_accrued += referrer_rebate;
         self.market_state.pc_fees_accrued += net_fees;
-        self.market_state.pc_deposits_total -= net_fees_before_referrer_rebate;
+        if !is_send_take {
+            self.market_state.pc_deposits_total -= net_fees_before_referrer_rebate;
+        }
```

Comment:
I don't understand why we need this, am I missing something?

Reply:
I think there exists a state where this value can underflow. I believe that `pc_deposits_total` is purely used for UI level aggregation and not important at all for actual logic (correct me if I'm wrong). The safest way to ensure that it's updated correctly in `SendTake` is to just set the values correctly after the orderbook has been processed, because we've eliminated the need for open orders accounting

Reply:
Yes true `pc_deposits_total` isn't relevant for the actutal logic. Can you point me to the code where the value gets updated instead after the orderbook has been processed?